packages/vx-xychart/src/components/grid/AnimatedGrid.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useContext } from 'react';
+import AnimatedGridRows from '@vx/react-spring/lib/grid/AnimatedGridRows';
+import AnimatedGridColumns from '@vx/react-spring/lib/grid/AnimatedGridColumns';
+import { AnimationTrajectory } from '@vx/react-spring/lib/types';
+import { CommonGridProps } from '@vx/grid/lib/types';
+import DataContext from '../../context/DataContext';
+
+export type AnimatedGridProps = {
+  rows?: boolean;
+  columns?: boolean;
+  animationTrajectory?: AnimationTrajectory;
+} & CommonGridProps;
+
+export default function AnimatedGrid({ rows = true, columns = true, ...props }: AnimatedGridProps) {
```

Comment:
might be able to pull out a `BaseGrid` to get rid of the common boilerplate between `Grid` + `AnimatedGrid`